overlord/fdestate/fdestate.go

```diff
@@ -191,6 +168,31 @@ type PrimaryKeyInfo struct {
 	Digest KeyDigest `json:"digest"`
 }
 
+type externalOperation struct {
+	Kind     string          `json:"kind"`
+	ChangeID string          `json:"change-id"`
+	Status   state.Status    `json:"status"`
```

Comment:
In theory fine, but the way the statuses are used suggest that we need a new type specific to this use-case